Modules/Forum/classes/class.ilObjForumNotificationDataProvider.php

```diff
@@ -486,6 +486,15 @@ public function getPosAuthorId()
 	public function getDeletedBy()
 	{
 		global $DIC;
-		return $DIC->user()->getLogin();
+
+		if($this->objPost->getUserAlias() && $this->objPost->getDisplayUserId() == 0
```

Comment:
Can you please extract this condition to a method with a meaningful name?

Reply:
done